membership/src/test/scala/zio/membership/transport/TcpTransportSpec.scala

```diff
@@ -33,8 +30,7 @@ object TransportSpec
               bind(addr).take(1).runCollect.map(_.head)
 
             environment >>> Live.live(for {
-              port   <- freePort
-              addr   <- SocketAddress.inetSocketAddress(port)
+              addr   <- SocketAddress.inetSocketAddress(0)
```

Comment:
Nice fix!

Reply:
Still fiddle around it, should be done tonight.